SCons/Environment.xml

```diff
@@ -1065,7 +1065,7 @@ Clean(['foo', 'bar'], 'something_else_to_clean')
 In this example,
 installing the project creates a subdirectory for the documentation.
 This statement causes the subdirectory to be removed
-if the project is deinstalled.
+if the project is uninstalled.
```

Comment:
A similar change is also in #4648

Reply:
Feel free to merge #4648 first, I'm OK to resolve the conflicts.